qml/FileListPage.qml

```diff
@@ -26,11 +26,21 @@ Page {
 
     property alias model: filteredModel.sourceModel
     property string title
-    property string searchText
+    property string searchText: searchField.text
+    property bool searchEnabled
     property QtObject provider
 
     allowedOrientations: Orientation.All
 
+    onSearchEnabledChanged: {
+        if (pageStack.currentPage.status == PageStatus.Active) {
+            (searchEnabled ? searchField : page).focus = true
```

Comment:
Tested with current 2.0.0.10, after choosing "show search" from the pulley, the searchfield appears, but not the virtual keyboard. I've checked that indeed searchField.focus is set to true on these lines, but maybe it's a forceActiveFocus() that should be called or something ?